src/service/ComposeDocument.ts

```diff
@@ -263,7 +263,7 @@ export class ComposeDocument {
                 // If the position is at the key separator, we're on the separator
                 pathParts.unshift(Sep);
                 pathParts.unshift(keyName);
-            } else if (params.position.character > indentLength) {
+            } else if (params.position.character >= indentLength) {

```

Comment:
This fixes an issue where the path was slightly incorrect, and hovering the mouse over the first character of a key did not correctly compute the path.